docs/integrations/spark.md

```diff
@@ -390,6 +390,28 @@ spark.hadoop.fs.lakefs.endpoint=‘https://example-org.us-east-1.lakefscloud.io
 
 For more details about [Mounting cloud object storage on Databricks](https://docs.databricks.com/dbfs/mounts.html).
 
+### Configuring Databricks SQL Warehouse with the S3-compatible API
+
+A SQL warehouse is a compute resource that lets you run SQL commands on data objects within Databricks SQL
+
+If you use Databricks SQL warehouse, you can take advantage of the lakeFS 
+S3-compatible API with the S3A FileSystem. 
+
+Define your SQL Warehouse configurations in the following way:
+
+1. In the top right, select `Admin Settings` and then `SQL warehouse settings`.
+
+2. Under `Data Access Configuration` add the following key-value pairs:
```

Comment:
What would be the style of URIs the user should use in this case? based on the configurations I guess that the scheme is lakefs//:. If this is the case let's add an example usage

Reply:
Good thing someone reads what I wrote here 😄